invenio_app_ils/ill/serializers/json.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2018 CERN.
```

Comment:
Wrong date, should we just change them all?

Reply:
thanks fixed